backend/tests/test_obj_views.py

```diff
@@ -0,0 +1,92 @@
+import json
+import pytest
+from io import BytesIO
+from rest_framework.test import APIClient
+from maskgen_api.models import Object, ObjectList
+
+pytestmark = pytest.mark.django_db  # ensures each test uses a test DB
+TEST_OBJ_FILE_PATH = "backend/tests/data/DCM5V5E.obj"
```

Comment:
This should use the relative path to the file, so it doesn't depend from where you run pytest.

```suggestion
import json
import os
import pytest
from io import BytesIO
from rest_framework.test import APIClient
from maskgen_api.models import Object, ObjectList

pytestmark = pytest.mark.django_db  # ensures each test uses a test DB
script_dir = os.path.dirname(__file__)
TEST_OBJ_FILE_PATH = os.path.join(script_dir, "data", "DCM5V5E.obj")
```